src/modules/config.ts

```diff
@@ -29,7 +29,7 @@ export default class Config extends API {
   async get(path?: string) {
     const cleanPath = path ? `/${path.replace(/\./g, '/')}` : ''
     const response = await this.sendGet(`config${cleanPath}`)
-    return response.json() as Promise<object>
+    return response.json() as Promise<object | string>
```

Comment:
what's going to cause the ambig here?

Reply:
well that's weird? didn't my comment come through?
> this call will return an object or a string... technically any valid JSON

Reply:
okay, makes sense... but why is the config coming back as json _or_ string? don't we control that?

Reply:
it depends what you ask for... if you ask for `config.get('Account.Thread')` you'll get that specific string, if you asked for `config.get('Account')` you'd get the whole object